tests/bson/bson-utcdatetime_error-001.phpt

```diff
@@ -1,22 +1,19 @@
 --TEST--
-BSON BSON\UTCDateTime #001 error
---INI--
-date.timezone=America/Los_Angeles
+BSON BSON\UTCDateTime requires object argument to implement DateTimeInterface
 --SKIPIF--
-<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM handles parameter parsing differently"); ?>
```

Comment:
I don't think it will be a problem. Your constructor accepts a mixed type and all validation is left to you, so it should be trivial to customize the exception message here.